# default setting for serial port, baudrate, motor min id, motor max id,
device_name_: "/dev/ttyUSB0"
baud_rate_: 1000000
protocol_version_: 2.0
